www/firebase.js

```diff
@@ -144,6 +144,11 @@ exports.setAnalyticsCollectionEnabled = function (enabled, success, error) {
     exec(success, error, "FirebasePlugin", "setAnalyticsCollectionEnabled", [enabled]);
 };
 
-exports.verifyPhoneNumber = function (number, success, error) {
-    exec(success, error, "FirebasePlugin", "verifyPhoneNumber", [number]);
+exports.verifyPhoneNumber = function (number, timeout, success, error) {
```

Comment:
This change is not backwards compatible.  If `timeout` is a function, the method is being called with the old signature.  If `timeout` is not a function, the method is being called with the new signature.